time/src/main/java/net/jqwik/time/internal/properties/arbitraries/DefaultZoneOffsetArbitrary.java

```diff
@@ -0,0 +1,112 @@
+package net.jqwik.time.internal.properties.arbitraries;
+
+import java.time.*;
+import java.util.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.arbitraries.*;
+import net.jqwik.time.api.arbitraries.*;
+
+import static org.apiguardian.api.API.Status.*;
+
+@API(status = INTERNAL)
+public class DefaultZoneOffsetArbitrary extends ArbitraryDecorator<ZoneOffset> implements ZoneOffsetArbitrary {
+
+	public static final ZoneOffset DEFAULT_MIN = ZoneOffset.ofHoursMinutesSeconds(-12, 0, 0);
+	public static final ZoneOffset DEFAULT_MAX = ZoneOffset.ofHoursMinutesSeconds(14, 0, 0);
+
+	private ZoneOffset offsetMin = DEFAULT_MIN;
+	private ZoneOffset offsetMax = DEFAULT_MAX;
+
+	@Override
+	protected Arbitrary<ZoneOffset> arbitrary() {
+
+		ZoneOffset[] zoneOffsets = generateAllValues();
```

Comment:
I suggest you just generate the number of 15-minute-intervals, positive and negative, and calculate the real offset in a final map operation.